docs/cli-propose.md

```diff
@@ -21,37 +23,73 @@ The following steps occur when this command is run:
 Using `npx`:
 
 ```
-npx sphinx propose <SCRIPT_PATH> --networks <testnets|mainnets> [options]
+npx sphinx propose <SCRIPT_PATH> --networks <NETWORK_NAMES...|testnets|mainnets> [options]
 ```
 
 Using `pnpm`:
 
 ```
-pnpm sphinx propose <SCRIPT_PATH> --networks <testnets|mainnets> [options]
+pnpm sphinx propose <SCRIPT_PATH> --networks <NETWORK_NAMES...|testnets|mainnets> [options]
 ```
 
 ### Parameters
 - `<SCRIPT_PATH>`: **Required**. The path to the Forge script file to propose.
 
 ### Options
-- `--networks <testnets|mainnets>`: **Required**. Choose between proposing on the test networks (`sphinxConfig.testnets`) or the production networks (`sphinxConfig.mainnets`) in your script.
+- `--networks <NETWORK_NAMES...|testnets|mainnets>`: **Required**. The network(s) to propose on. Options include:
+  - Arbitrary network names (e.g., `ethereum optimism arbitrum`): Propose on one or more networks, which must match the network names in the `rpc_endpoints` section of your `foundry.toml`.
+  - `testnets`: Propose on all of the test networks in your `foundry.toml`. Requires additional configuration; see the section titled [Proposing on all Test Networks or Production Networks](#proposing-on-all-test-networks-or-production-networks).
+  - `mainnets`: Propose on all of the production networks in your `foundry.toml`. Requires additional configuration; see the section titled [Proposing on all Test Networks or Production Networks](#proposing-on-all-test-networks-or-production-networks).
 - `--confirm`: **Optional**. Confirm the proposal without previewing it. Useful for automating proposals.
 - `--dry-run`: **Optional**. Perform a trial run without sending data to Sphinx's backend. Useful for testing and validation.
 - `--silent`: **Optional**. Suppress output to display only error messages. Must be combined with `--confirm` to confirm the proposal without previewing it.
 - `--target-contract <TARGET_CONTRACT>` (Alias: `--tc <TARGET_CONTRACT>`): **Optional**. Specify a contract in multi-contract scripts.
 
+## Advanced Usage
+
+### Proposing on all Test Networks or Production Networks
+
+To clearly differentiate between test networks and production networks in your `foundry.toml`, you must label the networks in its `rpc_endpoints` section. This involves inserting specific comment lines before and after the networks you want to categorize.
```

Comment:
You jump straight into explaining why the comment is necessary before actually covering the feature itself. I found this confusing even having read the info above on the `testnets` and `mainnets` keywords. 

We should add a short sentence here just giving brief overview of the feature. Something like this would be good:
> To more easily propose on a large number of networks simultaneously, you can use the `mainnets` or `testnets` keywords. These keywords are aliases for all of the networks you've labeled as `testnets` or `mainnets` in your `foundry.toml` file.

Reply:
Yeah that's much better